package.json

```diff
@@ -119,8 +118,6 @@
     "jsfuzz": "^1.0.15",
     "mitata": "^0.1.6",
     "mocha": "^10.0.0",
-    "mockttp": "^3.9.2",
```

Comment:
This is not actually used.
https://github.com/search?q=repo%3Anodejs%2Fundici+mockttp&type=code

Reply:
Thanks, I left it when working on the `RedirectHandler` and forget to remove it